src/nft/components/explore/CarouselCard.tsx

```diff
@@ -228,12 +239,12 @@ const MARKETS_ENUM_TO_NAME = {
   [Markets.LooksRare]: 'LooksRare',
 }
 
-export const CarouselCard = ({ collection, onClick }: CarouselCardProps) => {
+export const CarouselCard = ({ collection, href }: CarouselCardProps) => {
   const gqlCollection = useCollectionQuery(collection.address)
 
   return (
-    <CarouselCardBorder>
-      <CarouselCardContainer onClick={onClick}>
+    <CarouselCardBorder to={href}>
```

Comment:
nit: I feel like this component could now carry a better name (definitely with a `Link` at the end) to work better with a `to` prop.

Reply:
`CarouselLink`?